tests/ppx/react/expected/commentAtTop.res.txt

```diff
@@ -1,5 +1,6 @@
 type props<'msg> = {key?: string, msg: 'msg} // test React JSX file
 
+@react.component
```

Comment:
Something to test here is that this works well e.g. with generating interface files with the vscode extension.